lightning/src/ln/router.rs

```diff
@@ -941,17 +933,19 @@ impl Router {
 
 				for chan_id in $node.channels.iter() {
 					let chan = network.channels.get(chan_id).unwrap();
-					if chan.one_to_two.src_node_id == *$node_id {
-						// ie $node is one, ie next hop in A* is two, via the two_to_one channel
-						if first_hops.is_none() || chan.two_to_one.src_node_id != network.our_node_id {
-							if chan.two_to_one.enabled {
-								add_entry!(chan_id, chan.one_to_two.src_node_id, chan.two_to_one, $fee_to_target_msat);
+					if !chan.features.requires_unknown_bits() {
```

Comment:
Are you filtering out well first_hop and last_hop channel ? Not sure..

Reply:
last_hops no cause we call add_entry!() directly. first hops also no but because Features::relevant_init_flags_to_channel currently doesnt convert anything that is unknown (which should maybe be changed, but at least in this case it doesn't matter too much because we certainly wont stay connected if their Init message requires unknown bits).